README.md

```diff
@@ -1 +1,81 @@
-# widgets
+<h1 align='center' style='border-bottom: none;'>Webex Widgets</h1>
+<h3 align='center'>Library of React widgets to easily embed into your web applications!</h3>
+<p align='center'>
+<a href='https://circleci.com/gh/webex/widgets'>
+    <img alt='CircleCI' src='https://circleci.com/gh/webex/widgets.svg?style=shield'>
+  </a>
+  <a href='https://www.npmjs.com/package/@webex/widgets'>
+    <img alt='npm latest version' src='https://img.shields.io/npm/v/@webex/widgets?label=npm%40latest'>
+  </a>
+  <a href='#badge'>
+    <img alt='semantic-release' src='https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg'>
+  </a>
+  <a href='https://github.com/webex/widgets/blob/master/package.json#L28'>
+    <img src='https://img.shields.io/npm/l/webex.svg' alt='license'>
+  </a>
+</p>
+
+**Webex Components** is a set of [React](https://reactjs.org) widgets following Webex standard styling,
+aimed at react developers that want to embed the widgets into their applications.
+
+## Table of Contents
+
+- [Install](#install)
+- [Usage](#usage)
+- [Contributing](#contributing)
+- [License](#license)
+- [Support](#support)
+- [Team](#team)
+
+## Install
+
+```bash
+npm install --save @webex @webex/widgets
+```
+
+## Usage
+
+## Contributing
+
+We'd love for you to contribute to our source code and to make **Webex Widgets** even better than it is today! Here are some [guidelines](https://github.com/webex/widgets/blob/master/CONTRIBUTING.md) that we'd like you to follow.
+
+### Issues
+
+Please open an [issue](https://github.com/webex/widgets/issues) and we will get to it in an orderly manner.
+Please leave as much as information as possible for a better understanding.
+
+### Contributing Code
+
+We are using [Airbnb Style Guide eslint rule](https://github.com/airbnb/javascript) and [prettier](https://github.com/prettier/prettier) to lint the code style.
```

Comment:
Is there a reason for removing the "prettify" suggestion?

Reply:
Adam asked to not define that script in our package. So we don't have that anymore

Reply:
OK, we should still add a bit more, like "run the linter!" or something that explains more.
```suggestion
We are using [Airbnb Style Guide eslint rule](https://github.com/airbnb/javascript) and [prettier](https://github.com/prettier/prettier) to lint the code style. Make sure your code your code follows our lint rules before submitting!
```